app/assets/stylesheets/_design-system-waiting-room.scss

```diff
@@ -0,0 +1,15 @@
+// To be removed once design system incorporates styles included below.
+
+// basscss-utility-typography
+//------------------------------------------------
+.break-word {
+  word-wrap: break-word;
+}
+
+// could go in _util with .truncate-inline
```

Comment:
Comment will hopefully be invalid after [removal of mentioned style in #5953](https://github.com/18F/identity-idp/pull/5953/files#diff-200353e49446e5c851bd6528a9620231160d77300fafa8f0f928b135cc5091fb).

```suggestion
```